site/docs/v1/tech/identity-providers/index.adoc

```diff
@@ -53,3 +53,16 @@ When you enable an identity provider you're indicating that this external provid
 For each application, you can provide unique configuration. You might do this if you had two different FusionAuth applications that were both using Facebook as an identity provider, but using different Facebook applications. You can override none, some or all of the configuration values.
 
 Override settings are not available the "External JWT" identity provider, since you can create multiple instances of these.
+
```

Comment:
These parameters are also documented here if you want to link these sections. 
https://fusionauth.io/docs/v1/tech/oauth/endpoints See `idp_hint` and `login_hint`. 

Because FB and Google use modal dialogs, I do not believe these will work (currently). This was primarily added for SAML and OpenID Connect.  Twitter uses a redirect, but it is a different workflow I believe, so I do not know if this works. 

We would likely need to open an issue and do some testing with Twitter, FB and Google to see if and how it would work with these providers.

Reply:
OK, I think it is ok to just exclude these providers.